testdata/baselines/reference/submodule/compiler/declarationEmitLateBoundJSAssignments.js.diff

```diff
@@ -8,5 +8,8 @@
 -export namespace foo {
 -    let bar: number;
 -    let strMemName: string;
--}
-+export declare function foo(): void;
\ No newline at end of file
++export declare function foo(): void;
```

Comment:
Maybe not for this PR but I wonder what the leftover code difference is that's adding "declare" to these.